client/src/client.ts

```diff
@@ -67,7 +67,10 @@ export class Client {
         { scheme: "file", language: "erb" },
         { scheme: "file", language: "haml" },
         { scheme: "file", language: "slim" },
+        { scheme: "file", language: "blade" },
+        { scheme: "untitled", language: "blade" },
         { scheme: "file", language: "php" },
+        { scheme: "untitled", language: "php" },
```

Comment:
What's the difference between `scheme: "file"` and `scheme: "untitled"`? Is this for non-existent files?

Reply:
I saw it in another LSP setup. I _think_ it might be used for in-memory files by some clients, it's reference in [the spec](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#documentFilter) in a few places, but looks like it's not standard